### PR TITLE
final patch?

### PR DESCRIPTION
**Pull request by jane-lisy**
Eyyyyy I can't believe I can open a pull request on my own.
Changes:
- uploaded the necessary screenshots
- added part i
- added two corpora & two error classes to part iii

### CODE REVIEW

File: assign3.ipynb [hunk 1]
```diff
@@ -20,16 +20,81 @@
...
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "import nltk\n",
     "from nltk import load_parser, parse, grammar\n",
     "from nltk.draw.tree import draw_trees\n",
+    "import matplotlib\n",
```
Comment (dawnchandler) [inline assign3.ipynb]: Did your code use matplotlib?

Comment (dawnchandler) [inline assign3.ipynb]: Did your code use matplotlib?

File: assign3.ipynb [hunk 2]
```diff
@@ -20,16 +20,81 @@
...
+    "    'Usain Bolt broke the 100m record',\n",
+    "    'The Jamaica Observer reported that Usain Bolt broke the 100m record',\n",
+    "    'Andre said The Jamaica Observer reported that Usain Bolt broke the 100m record',\n",
+    "    'I think Andre said the Jamaica Observer reported that Usain Bolt broke the 100m record',\n",
+    "    'I shot an elephant in my pajamas',\n",
+    "    'I shoot beautiful pictures with my new camera',\n",
+    "    'he goes to the park every day',\n",
+    "    'he goes to the park to get fresh air'\n",
+    "]\n",
+    "sents = [nltk.word_tokenize(i) for i in sents]"
```
Comment (dawnchandler) [inline assign3.ipynb]: Can you replace the `i`s with `sent` or `s` for readability?

File: assign3.ipynb [hunk 3]
```diff
@@ -20,16 +20,81 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i in sents:\n",
+    "    for j in chart_parser.parse(i):\n",
```
Comment (dawnchandler) [inline assign3.ipynb]: Can you replace the `i`s in this for loop with `sent` or `s` and replace `j` with `tree`?

File: assign3.ipynb [hunk 4]
```diff
@@ -65,56 +130,37 @@
...
-      "  (VP[NUM='sg', TENSE='pres']\n",
-      "    (BE[NUM='sg', TENSE='pres'] is)\n",
-      "    (Adj[] precious))) \n",
-      "\n"
-     ]
-    }
-   ],
+   "outputs": [],
+   "source": [
+    "t_grammar = nltk.CFG.fromstring(\"\"\"\n",
```
Comment (dawnchandler) [inline assign3.ipynb]: I think you pasted your grammar into the wrong cell. It's in Part II instead of Part I.

File: assign3.ipynb (hunk 3 above)
Comment (dawnchandler) [inline assign3.ipynb]: Can you make this code runnable? I.e. load the grammar, make sure that chart_parser works.

File: assign3.ipynb [hunk 5]
```diff
@@ -576,6 +327,32 @@
     "    'To start the application process, you will need an updated CV or resume and a current unofficial or official transcript in English.',\n",
     "]\n",
     "\n",
+    "dps_sents = [\n",
```
Comment (dawnchandler) [inline assign3.ipynb]: Can you add the code to parse your sentences too? Just add `parse_and_draw(dps_sents)` and `parse_and_draw(mc_sents)` for both your datasets at the bottom of the code block, where mine are.

File: assign3.ipynb [hunk 6]
```diff
@@ -619,11 +397,26 @@
     "The parser performed very poorly with sentences that had parallelism as indicated by commas. In *Other scientists devote their time to fighting cancer, AIDS.*, the comma acts as a CC; *cancer* and *AIDS* should be on the same level. However, the parser has done something quite different and taken *fighting* as an verb that describes *cancer*. \n",
     "![Other scientists devote their lives to fighting cancer, AIDS.](./assets/partiii/other_scientists_devote.png)\n",
     "\n",
+    "##### Example Naming\n",
+    "When the sentence contains example naming, particularly when a verb or verb phrase is used as an example (which is generally in italics when read in the document), the parser assumes that the rest of the sentences is embedded within the example verb. For instance, the sentence *...then verbs like ~persuade~ become the unmarked case...*. *become...* becomes a VP complement to *persuade*, instead of being the VP that adjoins *verbs* to form S.\n",
+    "![image](./assets/partiii/example_naming.png)\n",
+    "\n",
+    "##### Conjunctions\n",
```
Comment (dawnchandler) [inline assign3.ipynb]: Do you think this should be combined with the CC Chunking subsection? I'm not sure.

Comment (jane-lisy) [reply assign3.ipynb]: I thought of that too, but I think adjective attachment is a bit different in the case that CC chunking involves multiple coordination of 3+ equal clauses (competition between logical connectives), whereas adjective attachment is concerned about the higher/lower attachment (competition with itself).

**Comment by dawnchandler** [comment]
Thanks for making the changes!

**Pull request merged**